renderer/components/Settings/SettingsFieldsSecurity.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { useFieldState, useFormState } from 'informed'
+import { useIntl } from 'react-intl'
+import { DataRow } from 'components/UI'
+import { PasswordInput, Toggle } from 'components/Form'
+import { FieldLabel } from './SettingsFieldHelpers'
+import messages from './messages'
+
+const SettingsFieldsSecurity = ({ currentConfig }) => {
+  const { value: isPasswordActive } = useFieldState('password.active')
+  const { submits } = useFormState()
+  const shouldValidateInline = submits > 0
```

Comment:
In some places it's `willValidateInline` is there any naming scheme behind these?

Reply:
We have typically have called this `shouldValidateInline`, however:

1) This fails our `react/boolean-prop-naming` lint rule
2) `will` is semantically more correct that `should` since the former is imperative statement rather than `should` which has a a probability associated with it.

I've added a refactor commit to convert all instances of `shouldValidateInline` to `willValidateInline`